SEED-platform/seed
# 1298 import date field

#### Any background context you want to provide?
Datetimes were not stored on when properties and taxlots were imported and/or modified. 

#### What's this PR do?
Add datetime stamps to the Property and TaxLot objects upon creation and modification. The tax lot data are stored on the Property/TaxLot tables and not the State tables. When a user updates the propertystate directly via the inventory detail page, then the database has a callback to update the Property modified time.

This PR also deprecated more API v1 around columns.

#### How should this be manually tested?
Import some data. Import another file with some modifications. Make sure that the created and updated fields change correctly in the inventory list. 

Also, try editing a detail property and taxlot and make sure that the updated datetime changes correctly.

- [x] @Myoldmopar -- in /api/v2/properties/{id}/, the id is the Property.pk, right? (not PropertyView). 

#### What are the relevant tickets?
#1298 

#### Screenshots (if appropriate)
The roll-ups look a little funny right now.
![screen shot 2017-12-04 at 1 30 48 pm](https://user-images.githubusercontent.com/1907354/33582440-79f8f590-d912-11e7-9fd6-c3055248b7d0.png)

#### Definition of Done:
